### PR TITLE
Tests start before feeds are updated

### PR DESCRIPTION
**Pull request by santipadilla**
|Related issue|
|-------------|
|    #5173         |

## Description

This PR changes the function that waits for the feed to finish. We achieve this by changing the log that tells us that the feed has finished and increasing the timeout.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -23,6 +23,7 @@ All notable changes to this project will be documented in this file.
 
 ### Changed
 
+- Change in the wait until the VD feed is completed ([#5227](https://github.com/wazuh/wazuh-qa/pull/5227)) \- (Tests)
```
Comment (Rebits) [inline CHANGELOG.md:26]:
```suggestion
- Change Vulnerability Detection feed updated waiter ([#5227](https://github.com/wazuh/wazuh-qa/pull/5227)) \- (Tests)
```

**Review by Rebits — CHANGES_REQUESTED** [review]
GJ, minor change is required

**Review by Rebits — APPROVED** [review]
LGTM

**Review by Rebits — CHANGES_REQUESTED** [review]
It seems after testing, that `INFO: Feed update process completed` log does not always appear in the manager.
When feeds are already downloaded, this log does not appear. This makes it impossible to test in an already prepared environment.

**Review by Rebits — APPROVED** [review]
LGTM

**Review by juliamagan — APPROVED** [review]

**Pull request merged**